cfgov/unprocessed/css/enhancements/layout.less

```diff
@@ -219,6 +219,20 @@
     } )
 }
 
+
+/* topdoc
+  name: Bulleted list with flush left alignment
+  family: cf-layout
+  notes:
+    - "Add flush left styling for bulleted lists"
+  tags:
+    - cf-layout
+*/
+
+.list__flush-left {
+    padding-left: 1em;
```

Comment:
Ah ok, it should be moved to the base styles or enhancements and updated to 18px (converted to EMs) to [match CF](https://github.com/cfpb/capital-framework/pull/450/files#diff-7049f34d37b91f0fc0ebe1c025293fdeR465)